test/parallel/test-fs-read.js

```diff
@@ -11,12 +11,12 @@ fs.read(fd,
         expected.length,
         0,
         'utf-8',
-        common.mustCall(function(err, str, bytesRead) {
+        common.mustCall((err, str, bytesRead) => {
           assert.ok(!err);
```

Comment:
This could be `assert.ifError(err);`, which gives slightly more information when anything fails.